test/commands/force/org/display.test.ts

```diff
@@ -0,0 +1,244 @@
+    .it('includes correct rows in non-json (table) mode', (ctx) => {
+      const result = ctx.stdout;
+      expect(result).to.include('Access Token');
+      expect(result).to.include('Connected Status');
+      expect(result).to.include('Client Id');
+      expect(result).to.include('Instance Url');
+      // not without verbose
+      expect(result).to.not.include('Sfdx Auth Url');
+    });
+
+  test
+    .do(async () => {
+      await prepareStubs({
+        refreshToken,
+      });
+    })
+    .stdout()
+    .command(['force:org:display', '--targetusername', 'nonscratch@test.com', '--verbose'])
+    .it('includes correct rows in non-json (table) mode with verbose', (ctx) => {
+      const result = ctx.stdout;
+      expect(result).to.include('Sfdx Auth Url');
+    });
+
+  test
+    .do(async () => {
+      await prepareStubs({
+        username: 'username@noalias.test',
+      });
+    })
+    .stdout()
+    .command(['force:org:display', '--json', '--targetusername', 'noalias@test.com'])
+    .it("omits alias when alias doesn't exist for username", (ctx) => {
+      const result = JSON.parse(ctx.stdout).result;
+      expect(result.alias).to.be.undefined;
+    });
+
+  test
+    .do(async () => {
+      await prepareStubs({
+        password: 'unEncrypted',
+      });
+    })
+    .stdout()
+    .command(['force:org:display', '--json', '--targetusername', 'nonscratch@test.com'])
+    .it('displays decrypted password if password exists', (ctx) => {
+      const result = JSON.parse(ctx.stdout).result;
+      expect(result.password).to.equal('unEncrypted');
+    });
+
+  test
+    .do(async () => {
+      await prepareStubs({
+        devHubUsername: devHub.username,
+      });
+      stubMethod($$.SANDBOX, Org, 'create').resolves(Org.prototype);
+      stubMethod($$.SANDBOX, Org.prototype, 'getOrgId').resolves(devHub.id);
+      stubMethod($$.SANDBOX, Org.prototype, 'getDevHubOrg').resolves({
+        getUsername: () => devHub.username,
+      });
+      stubMethod($$.SANDBOX, Org.prototype, 'getConnection').returns(Connection.prototype);
+
+      stubMethod($$.SANDBOX, Connection.prototype, 'sobject').returns({
+        find: async () => {
+          return [
+            {
+              Status: 'Active',
+              ExpirationDate: '2021-01-23',
+              CreatedBy: { Username: devHub.username },
+              Edition: 'Developer',
+              OrgName: 'MyOrg',
+              CreatedDate: '2020-12-24T15:18:55.000+0000',
+            },
+          ];
+        },
+      });
+    })
+    .stdout()
+    .command(['force:org:display', '--targetusername', 'scratch@test.com', '--json'])
+    .it('queries server for scratch org info', (ctx) => {
+      // console.log(ctx.stdout);
+      const result = JSON.parse(ctx.stdout).result;
+      expect(result).to.not.be.undefined;
+      expect(result.status).to.equal('Active');
+    });
+
+  it('gets non-scratch org connectedStatus');
```

Comment:
Still being written?

Reply:
We left pending test in toolbelt.  I'd prefer to keep around for ideas of "things we might want to add later when there's time"  

I made the list of test idea, then got coverage up to a good level, and wanted to get some ideas on this given sprint deadlines + QA.

Do we want these to not exist at all?

Reply:
Just curious, I am ok with leaving pending tests in place.